plow.md

```diff
@@ -138,12 +139,16 @@ People feel more included when they're supported.
 ### Family and friends first, work second
 Long lasting relationships are the [rocks of life](https://www.youtube.com/watch?v=6_N_uvq41Pg) and come before work.
 
+### Opportunity For Team and Company Bonding
```

Comment:
Suggested text. 

```
### Celebrate accomplishments with offsite events

While being frugal about spending, we also make opportunities for celebrating our accomplishments and bonding with other team members over offsite events. Our ambition is to have an all-company annual offsite meeting in the coming years. 

Until that happens, we will make sure the people in closer geographic circles will have quarterly events. For instance, an offsite lunch in the first and third quarters, an offsite weekend outing in the second quarter, and half-day fun activities in the fourth quarter. 
```

Reply:
I totally forgot about this PR :(. I will surely update in the different file as suggested like funcorner.MD

Reply:
Updated in the same file.